Update OpenLayers 4 examples with updated layer addresses from landsat

tigreped opened the pull request:
The [EarthData Site](https://wiki.earthdata.nasa.gov/display/GIBS/GIBS+API+for+Developers#GIBSAPIforDevelopers-BulkDownloading) suggests a new path [REST endpoint] (https://gibs{a-c}.earthdata.nasa.gov/wmts/epsg3413/best/)

And is a list of products available in the docs but it is not very clear about what each of them do, for instance, layer ASTER_L1T_Radiance_Terrain_Corrected which suggests:

 "ResourceURL format="image/png" resourceType="tile" template="https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/ASTER_L1T_Radiance_Terrain_Corrected/default/{Time}/{TileMatrixSet}/{TileMatrix}/{TileRow}/{TileCol}.png"/"

For instance: (https://gibs-a.earthdata.nasa.gov/wmts/epsg3857/best/ASTER_L1T_Radiance_Terrain_Corrected/default/GoogleMapsCompatible_Level13/8/16/10.png)

```
var landsat = new ol.layer.Tile({
            source: new ol.source.XYZ({
              url:'//gibs-{a-c}.earthdata.nasa.gov/wmts/epsg3857/best/VIIRS_CityLights_2012/default/GoogleMapsCompatible_Level8/{z}/{y}/{x}.jpg'
            }),
            visible: false,
            name: 'landsat8VIIRS'
        });
        map.addLayer(landsat);
```

mike-mcgann committed:
Added Bing example

mike-mcgann committed:
Added Google Maps example

mike-mcgann committed:
Added OpenLayers 3 endpoint

mike-mcgann committed:
Changed to protocol-less URI for Bing to prevent 'Blocked Loading Mixed Content' error in Firefox

mike-mcgann committed:
Merge branch 'google-bing' of github.com:nasa-gibs/gibs-web-examples into google-bing

mike-mcgann committed:
Bing URI needs a parameter to load via https. Changed GIBS references to https for Bing and Google

mike-mcgann committed:
Merge branch 'google-bing'

Conflicts:
	deploy/gibs-web-examples.spec

mike-mcgann committed:
Bing/Google live demos are now coming soon, updated README to explain the actual error returned

mike-mcgann committed:
Removed coming soon in README

mike-mcgann committed:
In README, referenced outstanding pull request for XYZ layers in OL3

mike-mcgann committed:
Time slider demo for OpenLayers 2

mike-mcgann committed:
Updated page title for OL2 slider

mike-mcgann committed:
OpenLayers 3 time slider demo

mike-mcgann committed:
Same styles for OL2/OL3

mike-mcgann committed:
Time slider with Leaflet

mike-mcgann committed:
Updated README for time slider examples

mike-mcgann committed:
To version 1.3.0

mike-mcgann committed:
Linting for JS/CSS

mike-mcgann committed:
Clean target

mike-mcgann committed:
Travis config

mike-mcgann committed:
Added Travis indicator

mike-mcgann committed:
Link to Worldview now at github

mike-mcgann committed:
Fixed typo in OL3 version

mike-mcgann committed:
Upgrade to OpenLayers 3 release version

mike-mcgann committed:
Updated README to put OL3 at top and fixed URLs

mike-mcgann committed:
openlayers/ol3#1124: Removed comment about XYZ being hard-coded for EPSG:3857

mike-mcgann committed:
Name OpenLayers 3 as OpenLayers

mike-mcgann committed:
More OL3 to OL changes

mike-mcgann committed:
Renamed openlayers3 to openlayers. Attempting to change WMTS to XYZ but waiting on openlayers/ol3@ec00cd32225ad44ed286e3d3bbeca995b7c4d74e to land in a release

mike-mcgann committed:
Endpoints with 512 tile size back to using WMTS

mike-mcgann committed:
Leaflet upgraded to 0.7.3

mike-mcgann committed:
To version 2.0

mike-mcgann committed:
Note about blank tiles at zoom level 0 with Web Mercator. Updated old OL2 links. Fixed zoom problem with geographic in OL3

mike-mcgann committed:
Updated apache config for new OL3 directory. Apache reloads on install

mike-mcgann committed:
CSS/JS lint reports

mike-mcgann committed:
WV-1068: Added Cesium to examples

mike-mcgann committed:
Updated README for Cesium

mike-mcgann committed:
In OL3, append TIME parameter to end of URL instead of overriding function

mike-mcgann committed:
Added some whitespace for readability

mike-mcgann committed:
To version 3.0.0

mike-mcgann committed:
Updated apache configuration

mike-mcgann committed:
Added cesium directory to RPM build

mike-mcgann committed:
Leaflet: maxZoom now explicit in geographic/time demos to prevent overzooming when using the zoom buttons

mike-mcgann committed:
Fixed links in README

mike-mcgann committed:
Removed protocol from URLs

mike-mcgann committed:
Upgrade to openlayers 3.1.1, proj4js 2.3.3

mike-mcgann committed:
Changing days in timeslider now throttled to one second, archive start date set to 2012

mike-mcgann committed:
Version bump

mike-mcgann committed:
Refactored directory structure for examples/demos and created a top level lib directory. Updated libraries as needed

mike-mcgann committed:
Added demos directory

mike-mcgann committed:
Initial sample version

mike-mcgann committed:
Initial version of Cesium demo

mike-mcgann committed:
Updated all GIBS links to developer page

mike-mcgann committed:
Added SST

mike-mcgann committed:
Added legends for cesium layers

mike-mcgann committed:
Merge pull request #2 from nasa-gibs/wv-1332.cesium

WV-1332: Cesium demonstration

mike-mcgann committed:
Moved GIBS geographic tiling scheme code into the lib directory. Updated Cesium demos to use EPSG:4326 by default

mike-mcgann committed:
Fixed link in README

mike-mcgann committed:
Point link to release version

mike-mcgann committed:
Updated source code links

mike-mcgann committed:
Upgraded to OpenLayers 3.4.0

mike-mcgann committed:
Add 20 degrees to start position

mike-mcgann committed:
Updated apache config, spec file for new examples location

mike-mcgann committed:
Adjust starting point due to globe view

mike-mcgann committed:
Merge branch 'master' of github.com:nasa-gibs/gibs-web-examples

mike-mcgann committed:
Version bump

mike-mcgann committed:
Updated to Cesium 1.9

mike-mcgann committed:
Updated Cesium version number in README

mike-mcgann committed:
Updated README with link to Cesium demo

rboller committed:
Upgraded examples to OL 3.15.1

rboller committed:
Fixed broken CSS link

rboller committed:
Updated bullet spacing

rboller commented:
Hi @tigreped, are you looking for scientific information about each of the layers?  If so, Worldview is providing them for many layers [here](https://github.com/nasa-gibs/worldview-options-eosdis/tree/master/common/config/metadata), for example for [ASTER GDEM](https://github.com/nasa-gibs/worldview-options-eosdis/blob/master/common/config/metadata/aster/GDEM.md).

Please let us know if you need further information.

ZachTRice committed:
Updated references to OpenLayers

Updated references to `OpenLayers` to explicitly say `OpenLayers 3` to prevent version confusion.

ZachTRice committed:
Changed references to Leaflet 0.7.x

Changed references to `Leaflet` 0.7.x to `Leaflet (Legacy)` as it is called on the Leaflet site.

ZachTRice committed:
Added placeholder for Bing API Key

ZachTRice committed:
Added placeholder for Google API Key

Commit:
hard coded https for gibs to work locally

ZachTRice committed:
Updated `openlayers` directory and references to `openlayers3`
- Updated to differentiate between ol3 and ol4.

ZachTRice committed:
Revert earthdata.nasa.gov link paths
